### DISCUSSION TITLE
What is your main/favorite web browser?

### DISCUSSION THREAD

**Discussion by Dendraspis** [staxrip/staxrip#443] [General]
I'm curious which web browser you use most or which one is your favorite. Tell me and maybe a quick *why*.  😊 

*And yes, I ask you video encoding folks on purpose!* 😉

**Comment by stax76** [comment]
I've always used Firefox until early 2020 were it wasn't usable any longer, every time I switched tabs, there was a 2-3 seconds lag, I tried everything, contacted support, nothing worked, so I started using Chrome. Meanwhile, the issue is solved, still happily using Chrome.

**Comment by Patman86** [comment]
I used Firefox and Chrome in parallel for a long time. Since the Edge Browser has been based on Chrome, I've been using it and I've been very satisfied for a long time.

Oh, why???
It's simple, clear and fast, just like you are used to from Chrome.

**Comment by JJKylee** [comment]
I've been using Chrome on all my devices since couple years ago: PC, Mac, iPad, and iPhone.
The main reason is because I can't live without Google's universal platform and services like YouTube, Google Drive, and so on.
And secondly, I don't wanna go through the hassle of setting up all the extensions I'm currently using on Chrome. 😓

Yeah, I admit. I'm Google's poor guinea pig. 😭

**Comment by Dendraspis** [comment]
@44vince44  I'm still waiting for your answer.  😉

**Reply by ghost** [reply]
Sorry I'm late (read below)

**Comment by ghost** [comment]
One browser is not enough.

- I use Chrome for the regular stuff, which I like to have on my phone as well (my phone is Android): news, youtube, etc
- I use Firefox for my serious stuff, like Github, but also for stuff I don't want to be traced for (like special conspiracy news, etc, yes I read ALL, and I don't believe ALL). Note:   In Firefox, I am not connected with my true identity. 
- I use Edge for online shopping only because the only online shopping site I use works better on Edge, but I could switch to Chrome anytime. But I won't, I'm liking Edge.
- I use Internet Explorer (yes you read well) to force some videos to download with a download manager (when it doesn't work on other browsers). Of course I need to disable the un-disablable MS add-on that forces a url opened in IE to open in Edge, which keeps reactivating itself at every update)

My favorite is Firefox, although it is not perfect.

**Comment by ZeroCool22** [comment]
FireFox Quantum all the way.

Reason?
The Fox is the real MVP!
